connection_manager.go

```diff
@@ -147,6 +153,9 @@ func (n *connectionManager) Run() {
 
 	for now := range clockSource {
 		n.HandleMonitorTick(now, p, nb, out)
+		memsetZero(p)
```

Comment:
`p` is never mutated, so this one is probably overkill (it's also a zero byte slice, so unless the compiler helps us out here, we would add the cost of a function call to do nothing).